doc/source/getting_started/faqs.rst

```diff
@@ -5,8 +5,8 @@ Frequently asked questions
 
 What is PyAnsys?
 ----------------
-PyAnsys is a set of open source technologies that allow you to interface Pythonically
-with Ansys Fluent, Mechanical APDL, AEDT, and other Ansys products. You can use PyAnsys
+PyAnsys is a set of open source technologies that allow you to interface with Ansys
+Fluent, Mechanical APDL, AEDT, and other Ansys products via Python. You can use PyAnsys
```

Comment:
```suggestion
Fluent, Mechanical APDL, AEDT, and other Ansys products with Python. You can use PyAnsys
```
Google dev doc style guide says not to use "via," probably because it's not as user-friendly for non-native English speakers. 
https://developers.google.com/style/word-list#via

Reply:
@PipKat maybe _using_ in order to avoid _...with...with..._?

Reply:
@PipKat  ...I'm closing this PR. We'll be creating doc PRs relative to a separate branch for some time, while we make big changes to the doc. This is because I would prefer to release the new version of the doc in one go.

On the subject of via, it turns out that there are many instances in our doc. We can look into trying to rephrase those.